src/lang/english.txt

```diff
@@ -2892,12 +2900,17 @@ STR_MAPGEN_BY                                                   :{BLACK}*
 STR_MAPGEN_NUMBER_OF_TOWNS                                      :{BLACK}No. of towns:
 STR_MAPGEN_DATE                                                 :{BLACK}Date:
 STR_MAPGEN_NUMBER_OF_INDUSTRIES                                 :{BLACK}No. of industries:
-STR_MAPGEN_MAX_HEIGHTLEVEL                                      :{BLACK}Maximum map height:
-STR_MAPGEN_MAX_HEIGHTLEVEL_UP                                   :{BLACK}Increase the maximum height of mountains on the map by one
-STR_MAPGEN_MAX_HEIGHTLEVEL_DOWN                                 :{BLACK}Decrease the maximum height of mountains on the map by one
-STR_MAPGEN_SNOW_LINE_HEIGHT                                     :{BLACK}Snow line height:
-STR_MAPGEN_SNOW_LINE_UP                                         :{BLACK}Move the snow line height one up
-STR_MAPGEN_SNOW_LINE_DOWN                                       :{BLACK}Move the snow line height one down
+STR_MAPGEN_HEIGHTMAP_HEIGHT                                     :{BLACK}Highest mountain:
+STR_MAPGEN_HEIGHTMAP_HEIGHT_UP                                  :{BLACK}Increase the maximum height of mountains on the map by one
+STR_MAPGEN_HEIGHTMAP_HEIGHT_DOWN                                :{BLACK}Decrease the maximum height of mountains on the map by one
```

Comment:
shouldn't these (and others) have a `_TOOLTIP` suffix?

Reply:
I think they all do, but given none had, I did not add it. I tried to keep the diff as small as I could, to prevent this PR blowing up any more than it already does.

There are a ton of these inconsistencies around mapgen. For example, sea-level is not in the settings. Sea level query has no own query caption. And indeed, several of these entries are missing `_TOOLTIP` etc.

I rather not fix it for this PR, but if we have some time, this sure could use a good code cleanup :)